projects/packages/videopress/src/class-block-editor-extensions.php

```diff
@@ -159,6 +159,7 @@ public static function enqueue_extensions() {
 			'jetpackVideoPressSettingUrl' => admin_url( 'admin.php?page=jetpack#/settings?term=videopress' ),
 			'isVideoPressModuleActive'    => Status::is_jetpack_plugin_and_videopress_module_active(),
 			'isStandaloneActive'          => Status::is_standalone_plugin_active(),
+			'imageURLBase'                => plugin_dir_url( __DIR__ ) . 'build/images/',
```

Comment:
really minor one:

```suggestion
			'imagesURLBase'               => plugin_dir_url( __DIR__ ) . 'build/images/',
```

if we commit this change, we'll have to update the client-side part.